spec/models/machine_spec.rb

```diff
@@ -5,4 +5,28 @@
     it { should validate_presence_of :location }
     it { should belong_to :owner }
```

Comment:
make sure to test that many to many on the machine model as well. Make sure that machines have many machine_snacks and they have many snacks through machine_snacks.

Reply:
No excuse for this one, simple oversight! Thank you for your comments